src/instructions/proctored_exam/ErrorProctoredExamInstructions.jsx

```diff
@@ -11,30 +10,27 @@ const ErrorProctoredExamInstructions = () => {
 
   return (
     <div>
-      <Container className="border py-5 mb-4">
-        <div className="h3">
-          <FormattedMessage
-            id="exam.ErrorProctoredExamInstructions.title"
-            defaultMessage="Error with proctored exam"
-          />
-        </div>
-        <p className="mb-0">
-          <FormattedMessage
-            id="exam.ErrorProctoredExamInstructions.text"
-            defaultMessage={'A system error has occurred with your proctored exam. '
-            + 'Please reach out to '}
-          />
-          <Hyperlink href={contactUsUrl}>
-            {platformName}
-          </Hyperlink>
-          <FormattedMessage
-            id="exam.ErrorProctoredExamInstructions.text"
-            defaultMessage={' for assistance, and return to the exam once you receive '
-            + 'further instructions'}
-          />
-        </p>
-      </Container>
-      <Footer />
+      <div className="h3">
+        <FormattedMessage
+          id="exam.ErrorProctoredExamInstructions.title"
+          defaultMessage="Error with proctored exam"
+        />
+      </div>
+      <p className="mb-0">
+        <FormattedMessage
+          id="exam.ErrorProctoredExamInstructions.text"
+          defaultMessage={'A system error has occurred with your proctored exam. '
+          + 'Please reach out to '}
+        />
+        <Hyperlink href={contactUsUrl}>
+          {platformName}
+        </Hyperlink>
+        <FormattedMessage
+          id="exam.ErrorProctoredExamInstructions.text"
```

Comment:
you already have the same id for the message above hyperlink, please change

Reply:
updated